simulation/simtypes/action.go

```diff
@@ -73,10 +73,22 @@ func (m msgBasedAction) Execute(sim *SimCtx, ctx sdk.Context) (
 	if err != nil {
 		return simulation.NoOpMsg(m.name, m.name, fmt.Sprintf("unable to build msg due to: %v", err)), nil, nil, nil
 	}
-	err = msg.ValidateBasic()
-	if err != nil {
-		return simulation.NoOpMsg(m.name, m.name, fmt.Sprintf("msg did not pass ValidateBasic: %v", err)), nil, nil, nil
+
+	// UNFORKING v2 TODO: Verify that type casting to this and then calling validateBasic is correct.
```

Comment:
.

Reply:
yeah, definitely ok to type cast that way.